### PR TITLE
Support running e2e tests for dev

### PR DESCRIPTION
**Pull request by ivan4th**
This change is [Reviewable](https://reviewable.io/reviews/mirantis/virtlet/444)

### CODE REVIEW

File: docs/devel/build-tool.md [hunk 1]
```diff
@@ -104,3 +104,11 @@ by CI.
 
 Starts the build container. This is done automatically by other
 `build/cmd.sh` commands that need the build container.
+
+## e2e
+
+Runs Virtlet e2e tests against the currently running DIND cluster.
+You can also pass test options to this command:
+```
+build/cmd.sh e2e -test.v -ginkgo.focus="Should have default route"
```
Comment (istalker2) [inline docs/devel/build-tool.md]: -ginkgo.focus="Should have default route"
why is this? The same for documentation

**Comment by ivan4th** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/devel/build-tool.md, line 113 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/444#-KvD3M8MQHgdnuXBJyyU-r1-113:-KvS7YB-Helf_Ko2tDo7:b-tyic4k) ([raw file](https://github.com/mirantis/virtlet/blob/6f191c66e5bfb24c8b361c67b71eae1ba1667e35/docs/devel/build-tool.md#L113)):*
<details><summary><i>Previously, istalker2 (Stan Lagun) wrote…</i></summary><blockquote>

 -ginkgo.focus="Should have default route"
why is this? The same for documentation
</blockquote></details>

It's meant to demonstrate that you can run just some e2e tests. Perhaps should add a note that it's optional.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/444)*

**Comment by ivan4th** [comment]
Review status: 1 of 3 files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/devel/build-tool.md, line 113 at r1](https://reviewable.io:443/reviews/mirantis/virtlet/444#-KvD3M8MQHgdnuXBJyyU-r1-113:-KvXav2CJeciIlEpUvmA:b-896fix) ([raw file](https://github.com/mirantis/virtlet/blob/6f191c66e5bfb24c8b361c67b71eae1ba1667e35/docs/devel/build-tool.md#L113)):*
<details><summary><i>Previously, ivan4th (Ivan Shvedunov) wrote…</i></summary><blockquote>

It's meant to demonstrate that you can run just some e2e tests. Perhaps should add a note that it's optional.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/444)*

**Comment by pigmej** [comment]
:lgtm:

---

Reviewed 1 of 3 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/444#-:-KvaGPLrkOArFUFajqEd:bnfp4nl)*

**Comment by istalker2** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/virtlet/444#-:-KvaHMWew2aBs0Z0zsyo:bnfp4nl)*

**Pull request merged**